breizorro/config/breizorro.yaml

```diff
@@ -0,0 +1,120 @@
+opts:
```

Comment:
opts should not be in the cab definition. Cab def only. Also, suggest placing this at top level, so it can be included as ``_include: (breizorro)breizorro.yml``.